cmd/mrCheckout.go

```diff
@@ -39,7 +39,7 @@ var checkoutCmd = &cobra.Command{
 		}
 		branch := mrs[0].SourceBranch
 		mr := fmt.Sprintf("refs/merge-requests/%s/head", mrIDStr)
-		gitf := git.New("fetch", targetRemote, mr, fmt.Sprintf(":%s", branch))
+		gitf := git.New("fetch", targetRemote, fmt.Sprintf("%s:%s", mr, branch))
```

Comment:
Possibly add a comment that links back to the git documentation? Or which man page shows you how to do this?